tony_georgiou/week_02/fansite/trivia.html

```diff
@@ -0,0 +1,78 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src="js/amazinggracefansite.js"></script>
+    <link rel="stylesheet" href="css/style.css">
+    <title>Grace Hopper's Trivial Pursuits</title>
+  </head>
+
+  <body>
+
+        <div class="nav">
+
+          <nav>
+            <ul>
+              <li><a href="index.html">Home</a></li>
+              <li><a href="biography.html">Biography</a></li>
+              <li><a href="hometown.html">Hometown</a></li>
+              <li><a href="trivia.html">Trivia</a></li>
+          </ul>
+         </nav>
+
+        </div>
+
+    <!--  <h1>A Date With Hopper</h1>
+      <p>If you were ever lucky enough to go on a dinner date with Rear Admiral Hopper you might have got  the chance to find out some of these amazing facts.
+      <h2>Amazing Data About "Amazing Grace"</h2>  -->
+
+<header>
+
+    <h2>Why We Will Always Admire Her:</h2>
+
+  <div class="cartoon">
+
+    <h2>Because she was the <span id="real">REAL</span> deal!</h2>
+    <img src="images/cobol.jpg" alt="funny cartoon of men and trex in office">
+
+  </div>
+
+</header>
+
+  <div class="trivia">
+
+    <ol>
+      <li>* At the tender age of seven she dismantled seven clocks trying to find out how their alarms worked.</li><br>
+      <li>* Applied for early entry to Vassar College to study Mathemtaics and Physics at the age of 16. She was refused entry, but was admitted the following year. She later continued her studies at Yale.</li><br>
+      <li>* At Harvard she worked on one of the very first mainframe computers ever built. It was 51 feet long, 8 feet high and 8 feet long. Now the same amount of computer memory would take up a space smaller than pin head.</li>,<br>
+      <li>* In 1959 she was part of a committee that defined the then new computer language, COBOL (Common Business-Oriented Language).</li><br>
+      <li>* She coined the phrase "debugging" after physically removing a moth that had got stuck in the university. That moth is still preserved at the Smithsonian Institute!</li><br>
+      <li>* She joined the US Navy at the age of 37 to help in the fight against Nazism.</li><br>
+      <li>* Grace spent over 43 years in the navy. She retired at the age of sixty, but was called back into service.</li><br>
+      <li>* She left the navy with the rank of Rear Admiral Lower (formerly known as Commodore). Thatis just two ranks below a full admiral.</li><br>
+      <li>* There is a US navy ship named in her honour, the USS Hopper.</li><br>
+      <li>* She is still known as the "Queen of Software."
+
```

Comment:
You shouldn't need to add the `*` marks yourself here. The `<li>` tag should do that for you. By default it uses a small filled circle, but you can change it using CSS code like this: 
```css
ul li:before {
		 content: '@';
```